lib/sitemap.xml

```diff
@@ -37,7 +37,7 @@
     </url>
   {% endunless %}{% endfor %}
   {% endunless %}{% endfor %}
-  {% for file in site.html_files %}
+  {% for file in site.sitemap_files %}
```

Comment:
Maybe we should just grab the namespace (e.g., `site.sitemap.files`) in case we need to inject other values down the line?